apps/client/src/app/pages/simulator/components/macro-popup/macro-popup.component.ts

```diff
@@ -40,10 +45,13 @@ export class MacroPopupComponent implements OnInit {
     this.macro = [[]];
     this.aactionsMacro = ['/aaction clear'];
     let totalLength = 0;
+    const reclaimBreakpoint = this.simulation.clone().run(true).simulation.lastPossibleReclaimStep;
     this.rotation.forEach((action) => {
+      console.log(reclaimBreakpoint);
```

Comment:
You left a console.log here ;)

Reply:
Ah, that was just to vet the code review process, making sure it's thorough! Like the programming version of a secret shopper.

Fixed, and merged in staging, thanks!